src/Opdex.Platform.WebApi/Validation/LiquidityPools/LiquidityPoolFilterParametersValidator.cs

```diff
@@ -0,0 +1,30 @@
+using FluentValidation;
+using Opdex.Platform.Common.Extensions;
+using Opdex.Platform.Infrastructure.Abstractions.Data.Queries;
+using Opdex.Platform.Infrastructure.Abstractions.Data.Queries.LiquidityPools;
+using Opdex.Platform.WebApi.Models.Requests.LiquidityPools;
+using System.Text.RegularExpressions;
+
+namespace Opdex.Platform.WebApi.Validation.LiquidityPools
+{
+    public class LiquidityPoolFilterParametersTests : AbstractCursorValidator<LiquidityPoolFilterParameters, LiquidityPoolsCursor>
+    {
+        private static readonly Regex Regex = new Regex("^[0-9A-Za-z ]+$");
```

Comment:
Noticed that you changed it so it's using the static `Regex.IsMatch()`. The way it was before instantiating a static `Regex` object is preferable. While `Regex.IsMatch()` does internally create a `Regex` object and it does cache it, creating the `Regex` obj (compiled takes >100ms) will now be done during the first request rather than on startup. And from then on if it ever gets purged from the internal cache.